style.css

```diff
@@ -189,13 +203,44 @@ footer {
 	padding-bottom: 50px;
 }
 
+.under-construction {
+	width: 40%;
+}
+
+/*portfolio*/
```

Comment:
looks like theres a lot of empty lines in your code here, best to remove those along with the comments :)